src/components/ui/Review.js

```diff
@@ -0,0 +1,87 @@
+import styled from 'styled-components/macro'
```

Comment:
You can actually move Review to components/entry!

Reply:
The reason is because Review will only be used in the entry details page, and it's not really a base UI component